### PR TITLE
fix: don't show links if field was empty

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes https://github.com/fossasia/open-event-frontend/issues/6546

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by mariobehling — APPROVED** [review]
The strange thing is, I cannot see this error appearing anymore on the live site. If this PR ensures the error does not re-appear even better.

File: app/templates/components/events/view/overview/speaker-view.hbs [hunk 1]
```diff
@@ -21,7 +21,9 @@
                 {{else if (eq field.fieldIdentifier 'gender')}}
                     {{@speaker.gender}}
                 {{else if (or (eq field.fieldIdentifier 'website') (eq field.fieldIdentifier 'github') (eq field.fieldIdentifier 'facebook') (eq field.fieldIdentifier 'twitter') (eq field.fieldIdentifier 'instagram') (eq field.fieldIdentifier 'linkedin'))}}
-                    <a class="truncate" href={{get @speaker field.fieldIdentifier}} target="_blank" rel="noopener">{{get @speaker field.fieldIdentifier}}</a>
+                    {{#if (and (not-eq (get @speaker field.fieldIdentifier) 'https://github.com/') (not-eq (get @speaker field.fieldIdentifier) 'https://facebook.com/') (not-eq (get @speaker field.fieldIdentifier) 'https://instagram.com/') (not-eq (get @speaker field.fieldIdentifier) 'https://twitter.com/') (not-eq (get @speaker field.fieldIdentifier) 'https://linkedin.com/'))}}
```
Comment (iamareebjamal) [inline app/templates/components/events/view/overview/speaker-view.hbs:24]: Not good, why the links were being saved in the first place

**Review by iamareebjamal — CHANGES_REQUESTED** [review]

**Comment by maze-runnar** [comment]
now links will save correctly. And keeping `if` for previosly saved links.

**Comment by iamareebjamal** [comment]
No, remove them. That's faulty logic. I'll manually make previous links null

Also, I have to think about why I added this && condition and the test so that it doesn't break anything

**Comment by maze-runnar** [comment]
> No, remove them. That's faulty logic. I'll manually make previous links null

ok

**Comment by mariobehling** [comment]
Yes, the issue is actually that these links exist in the first place. And they also show up on the public profile actually.

Screenshot from 2021-02-01 12-08-04

**Comment by maze-runnar** [comment]
> Yes, the issue is actually that these links exist in the first place. And they also show up on the public profile actually.
> 
> Screenshot from 2021-02-01 12-08-04

after this PR it would be resolved. as links would be save as null if they are not entered

**Pull request merged**